PLUB/Sources/Views/Home/Cell/RegisterInterest/InterestTypeCollectionViewCell.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import UIKit
+
 import Then
 import SnapKit
```

Comment:
이것도 위와 같이 알파벳 순은 어떠신가요?
```suggestion
import UIKit

import SnapKit
import Then
```

Reply:
앗 알파벳순으로 변경하겠습니다 ㅎㅎ